src/suparnaNJune20/Assignment25/SumOFNoFromString.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Assignment 25 : 8th July 2020
+
+WAP to sum all the numbers in given string.
+Input: J7yu9y8h1h8j4b7j3jjb6
+Ouput: Sum of all numbers in String is: 53
+ */
+package suparnaNJune20.Assignment25;
+
+public class SumOFNoFromString {
+	void sumOfNo(String str) {
+		int sum = 0;
+		for (int index = 0; index < str.length(); index++) {
+			if (Character.isDigit(str.charAt(index)))
+				sum += Integer.parseInt(str.valueOf(str.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));